crates/lox_core/src/bodies/cio/s06/terms.rs

```diff
@@ -0,0 +1,118 @@
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  1.0, -1.0,  1.0, -8.0, 12.0,  0.0], sin_coeff:     0.24e-6, cos_coeff:  0.12e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  2.0,  0.0,  0.0,  0.0,  0.0,  0.0], sin_coeff:    -0.32e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  2.0,  0.0,  2.0,  0.0,  0.0,  0.0], sin_coeff:    -0.28e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [1.0,  0.0,  2.0,  0.0,  3.0,  0.0,  0.0,  0.0], sin_coeff:    -0.27e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [1.0,  0.0,  2.0,  0.0,  1.0,  0.0,  0.0,  0.0], sin_coeff:    -0.26e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  2.0, -2.0,  0.0,  0.0,  0.0,  0.0], sin_coeff:     0.21e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  1.0, -2.0,  2.0, -3.0,  0.0,  0.0,  0.0], sin_coeff:    -0.19e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  1.0, -2.0,  2.0, -1.0,  0.0,  0.0,  0.0], sin_coeff:    -0.18e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  0.0,  0.0,  0.0,  8.0,-13.0, -1.0], sin_coeff:     0.10e-6, cos_coeff: -0.05e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  0.0,  2.0,  0.0,  0.0,  0.0,  0.0], sin_coeff:    -0.15e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [2.0,  0.0, -2.0,  0.0, -1.0,  0.0,  0.0,  0.0], sin_coeff:     0.14e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  1.0,  2.0, -2.0,  2.0,  0.0,  0.0,  0.0], sin_coeff:     0.14e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [1.0,  0.0,  0.0, -2.0,  1.0,  0.0,  0.0,  0.0], sin_coeff:    -0.14e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [1.0,  0.0,  0.0, -2.0, -1.0,  0.0,  0.0,  0.0], sin_coeff:    -0.14e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  4.0, -2.0,  4.0,  0.0,  0.0,  0.0], sin_coeff:    -0.13e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  2.0, -2.0,  4.0,  0.0,  0.0,  0.0], sin_coeff:     0.11e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [1.0,  0.0, -2.0,  0.0, -3.0,  0.0,  0.0,  0.0], sin_coeff:    -0.11e-6, cos_coeff:  0.00e-6 },
+    Term{ fundamental_arg_coeffs: [1.0,  0.0, -2.0,  0.0, -1.0,  0.0,  0.0,  0.0], sin_coeff:    -0.11e-6, cos_coeff:  0.00e-6 },
+];
+
+#[rustfmt::skip]
+// @formatter:off (sometimes RustRover ignores the rustfmt skip)
+pub(super) const FIRST_ORDER: [Term; 3] = [
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  0.0,  0.0,  2.0,  0.0,  0.0,  0.0], sin_coeff: -0.07e-6, cos_coeff:  3.57e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  0.0,  0.0,  1.0,  0.0,  0.0,  0.0], sin_coeff:  1.73e-6, cos_coeff: -0.03e-6 },
+    Term{ fundamental_arg_coeffs: [0.0,  0.0,  2.0, -2.0,  3.0,  0.0,  0.0,  0.0], sin_coeff:  0.00e-6, cos_coeff:  0.48e-6 },
+];
+
+#[rustfmt::skip]
```

Comment:
I've seen that in this and #34  you turn off the formatter for these files. Does it make sense?

Reply:
I am not sure. I tried multiple sections and they all seem reasonable.

This specific one turns into:

```

pub(super) const SECOND_ORDER: [Term; 25] = [
    Term {
        fundamental_arg_coeffs: [0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0],
        sin_coeff: 743.52e-6,
        cos_coeff: -0.17e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 0.0, 2.0, -2.0, 2.0, 0.0, 0.0, 0.0],
        sin_coeff: 56.91e-6,
        cos_coeff: 0.06e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 0.0, 2.0, 0.0, 2.0, 0.0, 0.0, 0.0],
        sin_coeff: 9.84e-6,
        cos_coeff: -0.01e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 0.0, 0.0, 0.0, 2.0, 0.0, 0.0, 0.0],
        sin_coeff: -8.85e-6,
        cos_coeff: 0.01e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
        sin_coeff: -6.38e-6,
        cos_coeff: -0.05e-6,
    },
    Term {
        fundamental_arg_coeffs: [1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
        sin_coeff: -3.07e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 1.0, 2.0, -2.0, 2.0, 0.0, 0.0, 0.0],
        sin_coeff: 2.23e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 0.0, 2.0, 0.0, 1.0, 0.0, 0.0, 0.0],
        sin_coeff: 1.67e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [1.0, 0.0, 2.0, 0.0, 2.0, 0.0, 0.0, 0.0],
        sin_coeff: 1.30e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 1.0, -2.0, 2.0, -2.0, 0.0, 0.0, 0.0],
        sin_coeff: 0.93e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [1.0, 0.0, 0.0, -2.0, 0.0, 0.0, 0.0, 0.0],
        sin_coeff: 0.68e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 0.0, 2.0, -2.0, 1.0, 0.0, 0.0, 0.0],
        sin_coeff: -0.55e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [1.0, 0.0, -2.0, 0.0, -2.0, 0.0, 0.0, 0.0],
        sin_coeff: 0.53e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 0.0, 0.0, 2.0, 0.0, 0.0, 0.0, 0.0],
        sin_coeff: -0.27e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0],
        sin_coeff: -0.27e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [1.0, 0.0, -2.0, -2.0, -2.0, 0.0, 0.0, 0.0],
        sin_coeff: -0.26e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [1.0, 0.0, 0.0, 0.0, -1.0, 0.0, 0.0, 0.0],
        sin_coeff: -0.25e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [1.0, 0.0, 2.0, 0.0, 1.0, 0.0, 0.0, 0.0],
        sin_coeff: 0.22e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [2.0, 0.0, 0.0, -2.0, 0.0, 0.0, 0.0, 0.0],
        sin_coeff: -0.21e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [2.0, 0.0, -2.0, 0.0, -1.0, 0.0, 0.0, 0.0],
        sin_coeff: 0.20e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 0.0, 2.0, 2.0, 2.0, 0.0, 0.0, 0.0],
        sin_coeff: 0.17e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [2.0, 0.0, 2.0, 0.0, 2.0, 0.0, 0.0, 0.0],
        sin_coeff: 0.13e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [2.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
        sin_coeff: -0.13e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [1.0, 0.0, 2.0, -2.0, 2.0, 0.0, 0.0, 0.0],
        sin_coeff: -0.12e-6,
        cos_coeff: 0.00e-6,
    },
    Term {
        fundamental_arg_coeffs: [0.0, 0.0, 2.0, 0.0, 0.0, 0.0, 0.0, 0.0],
        sin_coeff: -0.11e-6,
        cos_coeff: 0.00e-6,
    },
];
```

Reply:
Minor feel free to mark as resolved.